libs/styles/variables.css

```diff
@@ -97,6 +98,15 @@
   --type-detail-s-size: 10px;
   --type-detail-s-lh: 12px;
 
+  /* Code */
+  --type-code-all-weight: 400;
+  --type-code-l-size: 18px;
+  --type-code-l-lh: 27px;
+  --type-code-m-size: 16px;
+  --type-code-m-lh: 24px;
+  --type-code-s-size: 14px;
+  --type-code-s-lh: 21px;
```

Comment:
All line heights are actually `1.5` when compared to the fonts. Do we even need them defined explicitly?

Reply:
I did not notice that. I’m following the conventions of the surrounding typography, which do not follow a consistent scale. Should we be explicit like Consonant specs or are we free to optimize? I’m ok with either way.